### ISSUE TITLE
How to do node classification?

### ISSUE THREAD

**Issue by Yfhu1103** [HongyangGao/gunet#2]
Hello, I only see your code for graph classfication, But not for node classification, also, your data folder does not contain dataset like citeceer or cora, I want to know what's going on? Thank you for your reply.

**Comment by HongyangGao** [comment]
Hi, the code for node classification is written in TensorFlow. I may update them in a branch of this code base. Also this code can be applied to node classification by removing CNN parts.

**Comment by Mrslock** [comment]
Thank you for your reply, I carefully read your paper, and I find that in ops.py, you have fulfilled all the details and network structure for Graph u-net. I am unfamiliar with graph classification task, so could you please tell me what network.py is doing? If it is about graph classification, what method or network structure dose it refer to? Thanks.

**Comment by AppleXY** [comment]
Hello,I have seen that you said I need remove CNN layer to make node classification ,can you tell me the file name that contains the CNN parts?Thank you !

**Comment by Mrslock** [comment]
> Hello,I have seen that you said I need remove CNN layer to make node classification ,can you tell me the file name that contains the CNN parts?Thank you !

**Comment by Yfhu1103** [comment]
> Hello,I have seen that you said I need remove CNN layer to make node classification ,can you tell me the file name that contains the CNN parts?Thank you !

Hi, I have read the code and I think the CNN part is in "network.py", and the CNN architecture is according to the paper in AAAI 2018 "An End-to-End Deep Learning Architecture for Graph Classification", If you want node classification only, codes in "ops.py" is enough.

**Comment by psp3dcg** [comment]
Hello @Mrslock , have you succeeded training GraphUNet on cora dataset(node classification) with ops.py? I have tried modified the code, but the result is not so good. Training loss decreased but validation loss is not(floats a bit, almost constant).:(

**Comment by psp3dcg** [comment]
@HongyangGao , Hello, could you please release the graph unet code for node classification? Thanks a lot:)

**Comment by liuxy1103** [comment]
For node classification，how do I deal with the issue about batchsize?
